tmt/trying.py

```diff
@@ -192,7 +192,11 @@ def c(key: str) -> str:
                     {c('q')}{a('uit')}       clean up the run and quit the session
                 """))
 
-            answer = input("> ")
+            try:
+                answer = input("> ")
+            except EOFError:
+                return "quit"
```

Comment:
Unrelated to this particular change, but made me realize that we should be using an enum here :) Instead of returning a free-form, prone-to-typos string, `choose_action()` should return an enum one cannot make a mistake in. I.e. defining the set of actions, plus their descriptions and key to trigger them, and generating the help on line 180 from this list, as well as the return on line 203. If you try to sneak `{c('1')}{a('ogin')}` in one day, nobody is going to notice. `return TryActions.QUIT`? :)

Reply:
Nice idea! Implemented in f6ad7224. Please, review.